src/util/defaults.ml

```diff
@@ -179,7 +180,9 @@ let _ = ()
       ; reg Experimental "exp.privatization"     "'protection-read'" "Which privatization to use? none/protection-old/mutex-oplus/mutex-meet/protection/protection-read/protection-vesal/mine/mine-nothread/mine-W/mine-W-noinit/lock/write/write+lock"
       ; reg Experimental "exp.priv-prec-dump"    "''"    "File to dump privatization precision data to."
       ; reg Experimental "exp.priv-distr-init"   "false"  "Distribute global initializations to all global invariants for more consistent widening dynamics."
-      ; reg Experimental "exp.apron.privatization" "'mutex-meet'" "Which Apron privatization to use? dummy/protection/protection-path/mutex-meet"
+      ; reg Experimental "exp.apron.privatization" "'mutex-meet-tid-cluster12'" "Which apron privatization to use? dummy/protection/protection-path/mutex-meet"
```

Comment:
This description has an outdated list of privatizations now.

Also, do we want to make `tid-cluster12` the default then. The default base privatization is one without thread IDs though.

Reply:
I guess we should just make regular `mutex-meet` the default then.